js/main.js

```diff
@@ -13,236 +13,312 @@ const baseUrl =
     : "/";
 
 const BACKENDS = {
-  WebGPU: "WebGPU",
-  WebAssembly: "WebAssembly",
-  WebNN: "WebNN"
+  WASM: "WASM",
+  WEBGPU: "WebGPU",
+  WEBNN: "WebNN"
 };
 
-const SUPPORTED_DEVICES = {
-  GPU: "GPU",
+const DEVICES = {
   CPU: "CPU",
+  GPU: "GPU",
   NPU: "NPU"
 };
 
+// To add a new sample, please put <id>.png and <id>.mp4 in the public/assets folder.
 let SAMPLES = [
+  // WASM
   {
-    title: "Phi-3",
-    desc: "A private and powerful AI chatbot that runs locally in your browser.",
-    poster: `${baseUrl}assets/phi3.png`,
-    sampleUrl: "./samples/phi3-webgpu/index.html",
-    model: "Phi-3-mini-4k-instruct",
-    webApi: [BACKENDS.WebGPU],
+    id: "wasm_image_to_text",
```

Comment:
id is added for unified names of picture and video. In addition, id starts with backend name now for better maintenance.